examples/gallery/3d_plots/scatter3d.py

```diff
@@ -17,9 +17,20 @@
 import pandas as pd
 import pygmt
 
-# Load sample iris data and convert 'species' column to categorical dtype
+# Load sample iris data
 df = pd.read_csv("https://github.com/mwaskom/seaborn-data/raw/master/iris.csv")
+# Convert 'species' column to categorical dtype
+# By default pandas assigns the categorical number code in an alphabetical
+# manner to the individual categories. For a non-alphabetical order, you have
+# to adjust the categorical number code. For handling and manipulating
+# categorical data in pandas you may have a look at:
```

Comment:
For this paragraph this more general formulation should be fine. But maybe we should use ‘categories’ instead of ‘categorical values’, as this is the term used in the [pandas documentation]( https://pandas.pydata.org/docs/user_guide/categorical.html).